pkg/uptimerobot.go

```diff
@@ -112,6 +112,33 @@ func (a AlertContact) String() string {
 	return render(alertContactTemplate, a)
 }
 
+type Status int
+
+const (
+	Up        Status = 2
+	Down      Status = 9
+	MaybeDown Status = 8
+	Unknown   Status = 1
+	Paused    Status = 0
+)
+
+func (s Status) String() string {
+	switch s {
+	case Up:
```

Comment:
Would a map (of `[int]string`) be a nicer way to lay this out?

Reply:
Something like this?


```go
func (s Status) String() string {
	statuses := map[Status]string{
		Up:        "Up",
		Down:      "Down",
		MaybeDown: "Maybe Down",
		Unknown:   "Not Checked",
		Paused:    "Paused",
	}

	statusString, ok := statuses[s]
	if !ok {
		return "Unknown Status"
	}
	return statusString

}

```

Reply:
Yes, exactly!

Reply:
Updated!